components/user/ConfirmDeleteModal.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { Modal, View, Text, Button, StyleSheet } from 'react-native';
+
+interface ConfirmDeleteModalProps {
+    visible: boolean;
+    onConfirm: () => void;
+    onCancel: () => void;
+}
+
+const ConfirmDeleteModal = ({ visible, onConfirm, onCancel }: ConfirmDeleteModalProps) => {
+  return (
+    <Modal visible={visible} transparent animationType="slide">
```

Comment:
Can you switch to use the modal from `react-native-modal` library instead of the native one?